pkg/core/fluent_encoder.go

```
@@ -8,8 +8,8 @@
 // configuration. It wraps a standard zapcore.Encoder and is used to pass
 // configuration details implicitly.
 type FluentEncoder struct {
-	Config Configuration
 	zapcore.Encoder
```

golangci-lint (2.5.0)

[error] there must be an empty line separating embedded fields from regular fields

(embeddedstructfieldcheck)